clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -49,6 +49,7 @@ using namespace mlir;
 #include "clang/CIR/Dialect/IR/CIROpsDialect.cpp.inc"
 #include "clang/CIR/Interfaces/ASTAttrInterfaces.h"
 #include "clang/CIR/Interfaces/CIROpInterfaces.h"
+#include <clang/CIR/MissingFeatures.h>
```

Comment:
Curious this different inclusion style.
Anyway, I am compassionate about this blast effect https://github.com/llvm/clangir/pull/981/commits/21f97952e06926d3777e182b60cd901f34811e9b

Reply:
Yea, this looks off, unless everything changes to angle based, this should be quotes based in this PR.

Reply:
What exactly do you mean? I'm including MissingFeatuers here because it's the right thing to do to ensure we're guarding against the SSID. But I agree that it's really ugly. Hopefully we see some change from https://github.com/llvm/clangir/issues/1025!

Reply:
We mean just a minor
```suggestion
#include "clang/CIR/MissingFeatures.h"
```

Reply:
Good to go after you fix this one!